src/docs/asciidoc/Sources.asciidoc

```diff
@@ -113,6 +113,10 @@ customized using the `--mode` option:
 - *lines* Will split files line-by-line and emit a new message for each line
 - *contents* The default. Provides the contents of a file as a byte array
 
+When using `--mode=lines`, you can also provide the additional option `--splitWithMarkers=true`.
+If set to `true`, the underlying `FileSplitter` will emit additional _start-of-file_ and _end-of-file_ marker messages before and after the actual data.
+The payload of these 2 additional marker messages is of type `FileMarker`. The option `splitWithMarkers` defaults to `false` if not explicitly set.
```

Comment:
`FileMarker` is an inner class in the `FileSplitter` I find it as more useful info to specify inner classes with their outer in docs.